docs/getting-started/select-columns.md

```diff
@@ -0,0 +1,136 @@
+The CopyAction has a option called columnWhitelist which allows you to do just that.
+So we can simplify our action by replacing it with the following code:
+
+      select-airport-cols {
+        type = CopyAction
+        inputId = stg-airports
+        outputId = int-airports
+        columnWhitelist=["ident", "name", "latitude_deg", "longitude_deg"]
+        metadata {
+          feed = compute
+        }
+      }
+
+There are numerous other options available, which you can view in the [API Docs](http://smartdatalake.ch/docs/site/scaladocs/io/smartdatalake/workflow/action/CopyAction.html).
+
+
+
+## Try it out
+Note that we used a different feed this time that we called *compute*. 
+We will keep expanding the feed *compute* from now on.
+This allows us to keep the data we downloaded in the previous steps in our local files and just
+try out our new actions.
+
+To execute the pipeline, use the same command as before, but change the feed to compute:
+
+    docker run --rm -v ${PWD}/data:/mnt/data -v ${PWD}/config:/mnt/config smart-data-lake/gs1:latest --config /mnt/config --feed-sel compute
+
+Now you should see multiple files in the folder *data/int-airports*. Why is it split accross multiple files?
+This is due to the fact that the query runs with spark under the hood which computes the query in parallel for different portions of the data.
+We might work on a small data set for now, but keep in mind that this would scale up horizontally for large amounts of data.
+
+## More on Feeds
+
+SDL gives you precise control on which actions you want to execute. 
+For instance if you only want to execute the action that we just wrote, you can type
+
+    docker run --rm -v ${PWD}/data:/mnt/data -v ${PWD}/config:/mnt/config smart-data-lake/gs1:latest --config /mnt/config --feed-sel ids:select-airport-cols
+
+SDL also allows you to use combinations of expressions to select the actions you want to execute. You can run
+
+    docker run --rm smart-data-lake/gs1:latest --help
+
+to see all options that are available.
+
+One popular option is to use regular expressions to execute multiple feeds together.
+In our case, we can run the entire data pipeline with the following command : 
+
+    docker run --rm -v ${PWD}/data:/mnt/data -v ${PWD}/config:/mnt/config smart-data-lake/gs1:latest --config /mnt/config --feed-sel .*
+
+:::caution
+
+In our tutorial, this command will only work if you already have some files under *data/stg-airports* and data/stg-departures.
+This is because in the first step, we download files of which we don't know the schema in advance.
+The init-phase will require that for all dataObjects, the schema is known so that it can check for incompatibilities.
+When we already have some files, it will infer the schema based upon the files.
+One way to prevent this problem is to explicitly provide the schema for the JSON and for the CSV-File, 
+which is out of the scope of this tutorial.
```

Comment:
Can we show schema and schemaMin in a later part of the tutorial?

Reply:
Yes, I would put the to part 2 though